dfinity/sdk
# chore: updates config and templates for node 18 compatibility

# Description

Updates to package.json and the webpack config for Node 18

* Updates webpack packages
* changelog

Supports [SDK-422]

# How Has This Been Tested?

Manually tested - 

* nvm install 18
* nvm use 18
* dfx new
* dfx start --background
* dfx deploy
* npm start
* verify that calls from the frontend go through

# Checklist:

- [x] The title of this PR complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/).
- [x] I have edited the CHANGELOG accordingly.


[SDK-422]: https://dfinity.atlassian.net/browse/SDK-422?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
